lambda/analyzer.js

```diff
@@ -25,7 +25,13 @@ module.exports.handler = async(event, context) => {
         return console.log('[Dry-run] Skipping analysis');
     }
 
-    return findOptimalConfiguration(event);
+    const result = findOptimalConfiguration(event);
+
+    if (event.includeOutputResults) {
```

Comment:
It's a very minor change, mainly for consistency with the rest of the codebase, so I'd suggest something like:

``
if(!!event.includeOutputResults) {
``

just because I like to always evaluate booleans in if conditions 😄

Reply:
changed